unicode_io/inref/unicodefifo.m

```diff
@@ -29,12 +32,12 @@
 	set ^A=1
 	do test(file,"FIFO:WRITE:RECORDSIZE=-1","",FAIL)
 	set ^A=0
-	f i=1:1:60 quit:^A  hang 1
+	f i=1:1:900 quit:^A  hang 1
```

Comment:
There are 8 time loops being modified in this routine as well as the other. Only the second one exits (the 3rd exits regardless) if the loop runs too long. It that worth doing for the others so we aren't turning an 8 minute max test into a 2 hr or more max test?